src/ttkbootstrap/window.py

```diff
@@ -236,6 +233,11 @@ def _disabled_state_cursor(self, event):
         except:
             pass
         
+    def on_select_all(self, event):
```

Comment:
Let's add a private method indicator to be consistent with the disabled state method since this method is not really intended to be exposed in the public api. `_on_select_all(self, event):`

After that I think this is good to go.